### ISSUE TITLE
Update license to 2024

### ISSUE THREAD

**Issue by jvonderecken** [point8/data-science-learning-paths#7]
We should update all notebooks from to

```
---
_This notebook is licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)](https://creativecommons.org/licenses/by-nc-sa/4.0/). Copyright © 2018-2024 [Point 8 GmbH](https://point-8.de)_
```

**Comment by jvonderecken** [comment]
Maybe @clstaudt or @Vanessa-Mueller can take it. There are several ways (`find`, `rpl`, `sed`....). I did it once ;-)

**Comment by clstaudt** [comment]
Maybe GPT4  can take it:

```
I have a set of notebooks in a folder called notebooks as well as subfolders.

Each notebook has a copyright notice in the last cell:

This notebook is licensed under a [Creative Commons Attribution-NonCommercial-ShareAlike 4.0 International (CC BY-NC-SA 4.0)](https://creativecommons.org/licenses/by-nc-sa/4.0/). Copyright © 2018-2022 [Point 8 GmbH](https://point-8.de/)

I want a Python script that updates the notice to the current year. The script should be run from the command line.
```

-> https://github.com/point8/data-science-learning-paths/pull/8

**Comment by jvonderecken** [comment]
hehe...a bit more than a single line in the command line. But as long it works out.

**Comment by clstaudt** [comment]
Preferring Python as the only dependency.

**Comment by clstaudt** [comment]
Closed via #8

**Comment by jvonderecken** [comment]
👍
